### PR TITLE
[Merged by Bors] - chore: fix some cases in names

### PR DESCRIPTION
**Pull request by alexjbest**
And fix some names in comments where this revealed issues

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Limits/Cones.lean [hunk 1]
```diff
@@ -660,7 +660,7 @@ def functorialityEquivalence (e : C ≌ D) : Cocone F ≌ Cocone (F ⋙ e.functo
           by simp [← Equivalence.counitInv_app_functor]) }
 #align category_theory.limits.cocones.functoriality_equivalence CategoryTheory.Limits.Cocones.functorialityEquivalence
 
-/-- If `F` reflects isomorphisms, then `cocones.functoriality F` reflects isomorphisms
+/-- If `F` reflects isomorphisms, then `Cocones.functoriality f` reflects isomorphisms
```
Comment (alexjbest) [inline Mathlib/CategoryTheory/Limits/Cones.lean:663]:
```suggestion
/-- If `F` reflects isomorphisms, then `Cocones.functoriality F` reflects isomorphisms
```

File: test/toAdditive.lean [hunk 2]
```diff
@@ -131,7 +131,7 @@ run_cmd do
   let mul1 := `test.toAdditive._auxLemma |>.mkNum 1
   let mul2 := `test.toAdditive._auxLemma |>.mkNum 2
   let add1 := `test.toAdditive._auxLemma |>.mkNum 3
-  let add2 := `test.toAdditive._auxLemma |>.mkNum 4
+  let add2 := Additive._auxLemma |>.mkNum 4
```
Comment (alexjbest) [inline test/toAdditive.lean:134]:
```suggestion
let add2 := `test.toAdditive._auxLemma |>.mkNum 4
```

File: test/toAdditive.lean [hunk 3]
```diff
@@ -131,7 +131,7 @@ run_cmd do
   let mul1 := `test.toAdditive._auxLemma |>.mkNum 1
   let mul2 := `test.toAdditive._auxLemma |>.mkNum 2
   let add1 := `test.toAdditive._auxLemma |>.mkNum 3
-  let add2 := `test.toAdditive._auxLemma |>.mkNum 4
+let add2 := `test.toAdditive._auxLemma |>.mkNum 4
```
Comment (alexjbest) [inline test/toAdditive.lean:134]:
```suggestion
  let add2 := `test.toAdditive._auxLemma |>.mkNum 4
```

**Review by riccardobrasca — COMMENTED** [review]
Thanks a lot!! I wonder when this kind of things will be completely done...

bors merge

**Pull request closed without merging**